src/mmw/apps/modeling/tasks.py

```diff
@@ -195,7 +195,7 @@ def add_huc12(srat_huc12, aggregate):
                                             .next()['inputmod_hash'],
     }
 
-    aggregate['HUC-12s'] = {huc12_id: add_huc12(result, aggregate)
+    aggregate['HUC_12s'] = {huc12_id: add_huc12(result, aggregate)
```

Comment:
Elsewhere (database, cache keys, layer configuration) we just use `huc12` (without hyphen or underscore). Can we do that for simplicity? Like `HUC12s`?

Reply:
Yes, great idea. I'll remove the underscore before merging.